libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -197,12 +197,19 @@ - (id)contentFromData:(NSData*)data fromResponse:(NSHTTPURLResponse*)response er
                 network.securityToken = securityToken;
             }
         }
+    } else if (!content && !responseError && self.statusCodeToRaiseErrorWithEmptyResponse.count > 0) {
```

Comment:
Though I could see merit in having a way for the consumer to get a "first shot" at overriding error handling behavior in `CSFAction`.  In such a model, I would propose creating a block property on `CSFAction` that takes the content and response as an input, and returns an `NSError *` as output.  If set, you could put that in front of `CSFAction`'s call to `contentFromData:fromResponse:error:`.  The block would be responsible for returning an error if the response didn't pass its custom error handling, and `nil` if it didn't catch its custom errors.